### PR TITLE
Added the Email where required

### LINKED ISSUE #540: [CHANGE] CSPS page
### What page is the change request for? Please supply the URI.

https://a11y.canada.ca/en/canada-school-of-public-service-csps/

### Approval needed?

Yes, I need to approve this request before change goes live

### Yes, please send a request to the following person?

_No response_

### Who shall do the work?

- [ ] I will do it, I want to contribute
- [X] I am asking for someone else to please do the work

### Timeline

_No response_

### Additional context

Under the paragraphe Digitial Accessibility in Learning Team, the last sentence:

To learn more about inclusion and accessibility, or to access help with your learning design or development, please contact the Digital Accessibility **in learning** team.

Add the email address: digitalaccessibilityinlearning-accessibilitenumerique@csps-efpc.gc.ca

### PR DESCRIPTION
**Pull request by DhruvPatel2502**
## What does this pull request (PR) do? / Que fait cette demande « pull » (PR)?
Added the Email at the required place in https://a11y.canada.ca/en/canada-school-of-public-service-csps/ 

## Additional information (optional) / Information additionnelle (optionel)

**Related issues / Requêtes associées**
Fixes #540

### CODE REVIEW

File: src/pages/en/canada-school-of-public-service-csps.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ The CSPS Digital Accessibility in Learning team, under the User Experience divis
 - Provide coaching and professional development to School employees on accessibility.
 
 To learn more about inclusion and accessibility, or to access help with your learning design or development, please contact the Digital Accessibility team.
+ <a href="mailto:digitalaccessibilityinlearning-accessibilitenumerique@csps-efpc.gc.ca">digitalaccessibilityinlearning-accessibilitenumerique@csps-efpc.gc.ca</a>
```
Comment (shawnthompson) [inline src/pages/en/canada-school-of-public-service-csps.md:51]: Can you add a line to create a `p` element?

**Review by shawnthompson — CHANGES_REQUESTED** [review]
This change also needs to be done on the French page.

https://a11y.canada.ca/fr/ecole-de-la-fonction-publique-du-canada-efpc/index.html

File: src/pages/en/canada-school-of-public-service-csps.md (hunk 1 above)
Comment (DhruvPatel2502) [reply src/pages/en/canada-school-of-public-service-csps.md:51]: wrap the mail in p?

**Comment by shawnthompson** [comment]
Pages:

- https://deploy-preview-543--a11ycanada.netlify.app/en/canada-school-of-public-service-csps/
- https://deploy-preview-543--a11ycanada.netlify.app/fr/ecole-de-la-fonction-publique-du-canada-efpc/

**Review by shawnthompson — APPROVED** [review]

**Pull request merged**